stock_vertical_lift/models/vertical_lift_operation_pick.py

```diff
@@ -83,15 +85,30 @@ def _domain_move_lines_to_do_all(self):
     def fetch_tray(self):
         self.current_move_line_id.fetch_vertical_lift_tray_source()
 
+    def get_next_move_line(self, order):
+        def get_next(move_lines, current_move):
+            if not move_lines:
+                return False
+            move_lines_cycle = cycle(move_lines)
+            if not current_move:
+                return next(move_lines_cycle)
+            # Point to current_move and then return the next
+            while next(move_lines_cycle) != current_move:
+                continue
+            return next(move_lines_cycle)
```

Comment:
Nitpick, to avoid confusion between `stock.move` and `stock.move.line`
```suggestion
        def get_next(move_lines, current_move_line):
            if not move_lines:
                return False
            move_lines_cycle = cycle(move_lines)
            if not current_move_line:
                return next(move_lines_cycle)
            # Point to current_move_line and then return the next
            while next(move_lines_cycle) != current_move_line:
                continue
            return next(move_lines_cycle)
```

Reply:
Done